<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraUi</class>
 <widget class="QWidget" name="CameraUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1494</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>411</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Saved devices</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCopy">
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRename">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twSavedDevices">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Enable</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Camera name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Device</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Interface</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCamInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Camera information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>711</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlHwindows"/>
  </widget>
  <widget class="QGroupBox" name="Connecteddevices">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>291</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Connected devices</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>30</y>
      <width>281</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbFindConnectedDevices">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbInterface"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbSaveDevices">
         <property name="text">
          <string>Save device</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twConnectedDevices">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Device</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Interface</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbGrab">
         <property name="text">
          <string>Grab image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbLive">
         <property name="text">
          <string>Live</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblNoConnDevices">
         <property name="text">
          <string>Numer of devices: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>292</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pbInitializeDevices">
        <property name="text">
         <string>Initialize Devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbReinitializeDevices">
        <property name="text">
         <string>Reinitialize devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCloseAllDevices">
        <property name="text">
         <string>Close all devices</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListWidget" name="listWidgetInstanceDev"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
